keyboards/jd45/keymaps/via/rules.mk

```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+MIDI_ENABLE = no
```

Comment:
```suggestion
```
Technically this isn't needed as `rules.mk` stuff is case sensitive, and the `MIDI_ENABLE` at the keyboard level is `YES` instead of `yes`.
Might as well remove that too in this PR.